src/components/MobileWalletDialog/MobileWalletDialog.tsx

```diff
@@ -2,6 +2,8 @@ import { AnimatePresence } from 'framer-motion'
 import { MemoryRouter, Redirect, Route, Switch } from 'react-router'
 import { Dialog } from 'components/Modal/components/Dialog'
 
+import { CreateRouter } from './routes/CreateWallet/CreateRouter'
```

Comment:
nitpick: `CreateWalletRouter`

Reply:
Done in [4dc0d6b](https://github.com/shapeshift/web/pull/8760/commits/4dc0d6b0b7dce8f4017b70bf90e62cbb5efddf1e)